README.md

```diff
@@ -1,43 +1,159 @@
-# GeoMesa Ingest with Nifi
+GeoMesa NiFi Plugins
+====================
 
-This project contains three processors:
-* PutGeoMesa - Ingest data into GeoMesa with a GeoMesa converter or from geoavro
-* PutGeoTools - Ingest data into an arbitrary GeoTools Datastore based on parameters using a GeoMesa converter or avro
-* ConvertToGeoAvro - Use a GeoMesa converter to create geoavro
+NiFi manages large batches and streams of files and data.
+GeoMesa-NiFi allows you to ingest data into GeoMesa straight from NiFi by
+leveraging custom processors.
 
-To Setup...
+Installation
+------------
 
-Download and untar nifi 0.4.1 from the nifi website... https://www.apache.org/dyn/closer.lua?path=/nifi/0.4.1/nifi-0.4.1-bin.tar.gz
+### Install the Processors
 
-clone and build this repo...
+Pick a reasonable directory on your machine, and run:
 
-Copy the nar file (niagara files archive) geomesa-nifi/geomesa-nifi-nar/target/geomesa-nifi-nar-0.4.0-SNAPSHOT.nar into the $NIFI_HOME/lib/ directory
+```bash
+$ git clone https://github.com/geomesa/geomesa-nifi.git
+$ cd geomesa-nifi
+```
 
-For geomesa SFTs and configs install the gm-data-all resource bundle from the 
-[geomesa/gm-data](https://github.com/geomesa/gm-data) project. It contains converters and SFT specs for common types 
-such as twitter, geolife, osm-gpx, etc.
+To build, run
 
-start up nifi
-cd $NIFI_HOME
-bin/nifi.sh start
+```bash
+$ mvn clean install
+```
 
-tail -f logs/nifi-app.log
+To install the GoeMesa processors you will need to copy the geomesa-nifi-nar file from
+``geomesa-nifi/geomesa-nifi-nar/target/geomesa-nifi-nar-$VERSION.nar`` into the ``lib/`` directory of your NiFi installation.
 
-Go to http://localhost:8080/nifi
+### Install the SFTs and Converters
 
-Drag a GeoMesaIngestProcessor down...fill out the accumulo instance, etc...for sft and converter try "example-csv" and 
-copy an example csv from the geomesa-tools/examples/ingest/csv folder into a temp dir...use the getfile processor to 
-pick it up and route it to the PutGeoMesa...
+The GeoMesa processors need access to the SFTs and converters in order to ingest data. We can provide these to the
```

Comment:
I'd probably mention here that you can do one of two things....install this or just copy/paste them directly into the processor's configuration which is often easier if you dont' want to install or update the jar.